outputs/webhook.go

```diff
@@ -10,6 +10,12 @@ import (
 func (c *Client) WebhookPost(falcopayload types.FalcoPayload) {
 	c.Stats.Webhook.Add(Total, 1)
 
+	if len(c.Config.Webhook.CustomHeaders) != 0 {
+		for i, j := range c.Config.Webhook.CustomHeaders {
+			c.AddHeader(i, j)
+		}
+	}
```

Comment:
Compare to the following hunk in the current main branch:
https://github.com/falcosecurity/falcosidekick/blob/970a90504eb3d761c8cc9617a9153a9397ba5dbf/outputs/client.go#L187-L191